templates/templates/circleci/.circleci/config.yml.tmpl

```diff
@@ -20,6 +20,15 @@ jobs:
             - {{ $sshFingerprint }}
           {{- end }}
 {{- end }}
+      - run:
+          command: |
+            mkdir /tmp/terraform-docs
+            cd /tmp/terraform-docs
+            # https://terraform-docs.io/user-guide/installation/#pre-compiled-binary
+            curl -sSLo ./terraform-docs.tar.gz https://github.com/terraform-docs/terraform-docs/releases/download/v0.16.0/terraform-docs-v0.16.0-linux-amd64.tar.gz
+            tar -xzf terraform-docs.tar.gz
+            chmod +x terraform-docs
+            echo "$PWD" >> $GITHUB_PATH
```

Comment:
@alldoami I think GITHUB_PATH is a github action specific thing https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path . Getting an error when trying to update fogg :( https://app.circleci.com/pipelines/github/FB-PLP/terraform-infra-management/8174/workflows/1bc186e9-befd-4527-8c52-07a7f812128a/jobs/8183

I don't know if there's an equivalent for circleci?